### ISSUE TITLE
Transition from Mailchimp to Sendinblue

### ISSUE THREAD

**Issue by nathenf** [life-itself/community#346]
On the 5th of March, Mailchimp subscription options change and will therefore be costly for us ($40 a month). We will move to Sendinblue instead ($25 a month). 

Job story: We want to move to Send in Blue, so that we reduce our communication costs while maintaining a great email tool. 

## Acceptance

- [x] Sendinblue is set-up with contacts, segments, etc. ✅ 
- [x] Email sign-ups on website all link to sendinblue rather than Mailchimp ✅ **see #359** 
- [x] Email template is designed with branding [Link Here](https://my.sendinblue.com/camp/template/16/message-setup) ✅ 
- [x] Welcome email created for new subscribers ✅ **Welcome email will send when someone is added to the Life Itself General list. Need to follow up with Khalil to make sure people who sign-up through our website are automatically added. Worth revisiting the email in a few months to tweak copy relevant to our latest work**

## Tasks

- [x] Move contacts from Mailchimp to Sendinblue: We have account for both so just a matter of exporting and importing. Make a note of current audience at date of transfer. We will later need to amend any unsubscriptions or subscriptions. Also, lets not transfer any unsubscribed contacts or contacts that recently have bounced ✅ **In two lists "Life Itself General" & "Web3"
- [x] Create Issue for Khalil to update sign-up forms on website and provide API for this ✅ [Issue Link #359](https://github.com/life-itself/community/issues/359)
- [ ] Write a welcome email and automate ✅  **welcome email written here: https://docs.google.com/document/d/1XvuQvFKIfaGq-fHdbQa-yrk7qY5EB_tKddOYn8EoF5A/edit**
- [x] Move Newsletter Template from MailChimp to SendinBlue ✅ [Find Preview here](https://my.sendinblue.com/camp/preview/id/25)

**Comment by ZybN** [comment]
@nathenf I will need to sync with you since Mailchimp and SendinBlue accounts require a verification code.

**Comment by nathenf** [comment]
@rufuspollock accounts@lifeitself.org already has 7000+ contacts, I assume this is Datopian contacts. In which case should we start a new one with the comms email instead?

**Comment by ZybN** [comment]
- I have moved Mailchimp Subscribers to Send in Blue, The subscribers are in the List named Mailchimp Subscribers.

- I have downloaded the contacts from Send in Blue and created a backup of contacts from Mailchimp and Send in Blue in Google Drive: https://drive.google.com/drive/folders/1I_H6UnwtlkVxAMUsgyquiluQhf0JQhJ7?usp=share_linkvg

**Comment by nathenf** [comment]
Hey @ZybN, Great work! I had a look at the list Mailchimp Subscribers, doesn't include the segements from Mailchimp. We need to know which is the web3 segment and which is the general Life Itself segement

**Comment by nathenf** [comment]
@ZybN for this next task = Create Issue for Khalil to update sign-up forms on website and provide API for this. Can you create this issue and assign to Khalil. You should include the job story and any information Khalil might need. Let me know if you need any clarification

**Comment by ZybN** [comment]
> Hey @ZybN, Great work! I had a look at the list Mailchimp Subscribers, doesn't include the segements from Mailchimp. We need to know which is the web3 segment and which is the general Life Itself segement

Hi @nathenf I have edited the columns and now we can see the tags of "Web3" and "Life Itself General" but the format isn't like Mailchimp. I have created a segment of Web3 contacts but I am not able to develop Segments of other Tags. Because the tags are imported with additional information. 

Lets connect on a call and fix this.

**Comment by rufuspollock** [comment]
@nathenf @ZybN could we get a due date on this one (and i'd perhaps move the last acceptance criteria i.e. sending out the new newsletter to its own issue both b/c substantial and distinct and should not block completing this - i think you can probably test the setup with just a send out to one or two internal people if you want to do a test in advance).

**Comment by nathenf** [comment]
@rufuspollock currently waiting on Khalil to edit the website sign-up forms but he has been ill, will move newsletter and template building to another issue for this months newsletter

**Comment by nathenf** [comment]
@rufuspollock due date is this Friday. Just welcome email and transferring design over.

**Comment by nathenf** [comment]
FIXED - All contacts moved to Sendinblue, Template made for newsletter, welcome email set up and activated.

**Comment by nathenf** [comment]
@ZybN can you include the number of hours you worked on this in the actual hours data? I've added mine

**Comment by rufuspollock** [comment]
@nathenf @ZybN yay! 🎉 🛳️ 🚀 👏
